server/routes/splunk/splunk_routes.py

```
@@ -85,7 +86,11 @@ def _request(self, method: str, path: str, **kwargs) -> requests.Response:
         except requests.exceptions.SSLError as exc:
             # SSLError must come before ConnectionError (it's a subclass)
             logger.error(f"[SPLUNK] {method} {url} SSL error: {exc}")
-            raise SplunkAPIError("SSL/TLS error. Check the instance URL protocol (https://).") from exc
+            raise SplunkAPIError(
+                "SSL/TLS error connecting to Splunk. If your instance uses a self-signed "
+                "or internal CA certificate, set SPLUNK_SSL_VERIFY to a CA bundle path, "
+                "or to 'false' to disable verification (not recommended for production)."
+            ) from exc
```

Ruff (0.15.10)

[warning] Avoid specifying long messages outside the exception class

(TRY003)